farm/data_handler/data_silo.py

```diff
@@ -493,7 +492,38 @@ def __init__(self, processor, batch_size, dataloader_workers=8):
         :type dataloader_workers: int
         """
 
-        self.loaders = defaultdict(lambda: None)
+        self.processor = processor
+        self.batch_size = batch_size
+        self.dataloader_workers = dataloader_workers
+
+    def get_data_loader(self, dataset_name):
+        """
+        Returns a new instance of dataloader for the given dataset.
+
+        The dataloader lazily yields from Iterable DataSets. After a complete iteration 
+        of dataloader, the generators gets exhausted. So, for instance, in the case of
```

Comment:
"After a complete iteration over the input data, the generator gets exhausted" ?